msteams-platform/sbs-gs-javascript.yml

```diff
@@ -262,7 +262,6 @@ items:
     | - `.fx/configs/config.<envName>.json` | Configuration file for every environment. |
```

Comment:
The pic in the following page needs update
https://docs.microsoft.com/en-us/microsoftteams/platform/sbs-gs-javascript?tabs=vscode%2Cvsc%2Cviscode%2Cvcode&tutorial-step=2
(under Take a tour of the tab app source code)